### ISSUE TITLE
how can I achieve higher f1 value?

### ISSUE THREAD

**Issue by MingHong1019** [INK-USC/TriggerNER#9]
I run: python semi_supervised.py --device cuda:0 --dataset CONLL --percentage 20
but the result can only achieve  54.20(F1) in test set. Which command should I run can achieve over 80 F1 value?

**Comment by danny911kr** [comment]
Hi, @MingHong1019 

Thank you for reaching out to us. Your command is correct.
The exact meaning of your command is "using 4% of original training data with triggers",
since our trigger data is 20% of original training data with their triggers.

However, the correct test result should be somewhat between 77~80 F1 score.
I ran it in my environment and could achieve 78.64.

I think there was a bug in reading trigger data, so I fixed it.
Could you please try running it again? Thanks a lot.
